### ISSUE TITLE
Add support for <style> tags in head for inline style rendering. 

### ISSUE THREAD

**Issue by lxe** [nfl/react-helmet#136]
There are css components and libraries out there that render `<style>`s inside head. Making these libraries inter-operate with helmet make for some non-optimal server-only logic. 

Would it make sense to add support to `<style>` in Helmet? I'm guessing it would behave the same way `<script>` is behaving, except without the src attribute.

**Comment by cwelch5** [comment]
Thanks for the suggestion.  Got a PR up now - #140

**Comment by lxe** [comment]
@cwelch5 awesome! Thanks for this.

**Comment by KCraw** [comment]
Nice! 

Docs in readme are inaccurate though...

```jsx
<Helmet
    style={{
        "cssText": `
            body {
                background-color: green;
            }
        `
    }}
/>
```

... throws an error. Should be...

```jsx
<Helmet
    style={[{
        "cssText": `
            body {
                background-color: green;
            }
        `
    }]}
/>
```

**Comment by cwelch5** [comment]
@KCraw Thanks!  Fixed.

**Comment by cwelch5** [comment]
This has been released in react-helmet@3.1.0
